task/preflight/0.1/preflight.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preflight
+spec:
+  description: >-
+    Scans container images for certification readiness
+  params:
+    - name: image-url
+      description: Image url to scan.
+  results:
+    - name: TEST_OUTPUT
+      description: Preflight pass or fail outcome.
+    - name: PFLT_OUTPUT
+      description: Preflight check summary.
+  steps:
+    - name: check-container
+      image: quay.io/opdev/preflight:stable
```

Comment:
please pin image to digest
quay.io/opdev/preflight:stable@sha256:9a868eb7f2d6f4c5a14e0da03f9a2342ee726c62ba2f129d0ca29a302c5d67d6

Reply:
you omitted `stable` tag, please keep it there together with digest

Reply:
oops. added. thanks for the feedback 👍